impl/maven-executor/src/main/java/org/apache/maven/cling/executor/embedded/EmbeddedMavenExecutor.java

```diff
@@ -296,11 +306,18 @@ protected Context doCreate(Path mavenHome, ExecutorRequest executorRequest) {
                         try {
                             if (r.stdoutConsumer().isPresent()
                                     || r.stderrConsumer().isPresent()) {
+                                System.setProperty("org.jline.terminal.output", "out");
```

Comment:
We don't have JLine in the classpath, right ? Else `TerminalBuilder.PROP_OUTPUT` would be more appropriate.